packages/react-component-library/src/components/TabNav/TabNavItem.tsx

```diff
@@ -2,12 +2,16 @@ import React from 'react'
 
 import { NavItem } from '../../common/Nav'
 import { StyledTabItem } from '../TabBase/partials/StyledTabItem'
-import { StyledTab } from '../TabBase/partials/StyledTab'
+import { StyledTabNavTab } from './partials/StyledTabNavTab'
 
 export const TabNavItem: React.FC<NavItem> = ({ isActive, link }) => (
   <StyledTabItem data-testid="tab-nav-tab">
-    <StyledTab $isActive={isActive} data-testid="tab-nav-tab-button">
-      <div>{link}</div>
-    </StyledTab>
+    <StyledTabNavTab
+      as="div"
```

Comment:
The error here was:

https://dequeuniversity.com/rules/axe/4.4/nested-interactive

This was due to the `button` role containing a `link` role (same as the example in the link).

This change resolves the problem by changing the `<button>` element to be a `<div>`.